src/routes/api/rating.route.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import { protectRoute, restrictTo } from '../../middlewares/auth.middleware';
+import { getRatingsById, productRating } from '../../controllers/ratings.controller';
+
+import {checkIfSellerOwnsProduct, checkProductAvailable} from '../../middlewares/product.middleware';
+import { checkIfUserBoughtProduct, validateRating } from '../../middlewares/ratings.middleware';
+import { uuidValidation } from '../../validations/user/userId.validation';
+  const ratingRoute = express.Router();
+
+
+  ratingRoute.post('/',
+  protectRoute,
+  restrictTo('buyer', 'admin'),
+  validateRating,
+  checkProductAvailable,
+  checkIfUserBoughtProduct,
+  productRating
+);
+
+ratingRoute.get('/:productId',
+protectRoute,
+uuidValidation('productId'),
+checkIfSellerOwnsProduct,
+getRatingsById
+);
```

Comment:
- Change the endpoint as I showed you yesterday
- Do you know why you check if the seller owns a product? multiple users can view the product ratings. You don't need to restrict them